lxd/firewall/drivers/drivers_nftables.go

```diff
@@ -481,6 +481,17 @@ func (d Nftables) InstanceSetupProxyNAT(projectName string, instanceName string,
 	var dnatRules []map[string]interface{}
 	var snatRules []map[string]interface{}
 
+	targetPortRanges := PortRangesFromSlice(forward.TargetPorts)
+	for _, targetPortRange := range targetPortRanges {
+		targetPort := portRangeStr(targetPortRange, "-")
+		snatRules = append(snatRules, map[string]interface{}{
+			"ipFamily":   ipFamily,
+			"protocol":   forward.Protocol,
+			"targetHost": targetAddressStr,
+			"targetPort": targetPort,
```

Comment:
Could change this variable and template field name to indicate that it could contain multiple ports now.